tutorials/llm/llama-3/nemo2-sft-peft/nemo2-sft.ipynb

```diff
@@ -0,0 +1,632 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Learning Goals\n",
+    "\n",
+    "## Supervised Finetuning (SFT)\n",
```

Comment:
suggest a revised title to: 1) inform the user that this is a procedure and 2) improve SEO searches - we have a lot of content with PEFT in the title and we should differentiate its purpose here.

"## Optimizing Foundation Models with Supervised Fine-Tuning (SFT)\n",